sources/accessible-name-and-description.ts

```diff
@@ -83,6 +85,7 @@ function isHidden(
 	}
 
 	const style = getComputedStyleImplementation(node);
+	maybeNodeStyle.current = style;
```

Comment:
The function name indicates it's a pure getter but now has a side-effect. Shouldn't we instead leverage `getComputedStyleImplementation` to use a variant the caches the  way you intended?

Reply:
yeah, doing that is a lot nicer